package.json

```diff
@@ -23,6 +24,20 @@
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
+    "bootstrap": "^3.3.7",
```

Comment:
shouldn't mocha also be in the `dependencies` to allow for
`export NODE_ENV=production && npm i && npm test` to be able to run tests using mocha under production environment? Otherwise you would have to remember (and the CI system) to set `NODE_ENV` to `development` before `npm i`